### DISCUSSION TITLE
Any good issues or bugs that are good for new contributors?

### DISCUSSION THREAD

**Discussion by ClifHouck** [triton-inference-server/server#6330] [General]
I tried looking for some issues using the 'good first issue' label, but there don't seem to be any open ones with that label. Any recommendations for someone looking to contribute to the triton server?

**Comment by the-david-oy** [comment]
Thank you for wanting to contribute and asking this question!

Going through the issues could help. Especially the ones labeled "enhancement." The best ones to start with would be the ones that you are most familiar with, either due to using those features in Triton or your prior knowledge. For example, I see this [Prometheus feature request](https://github.com/triton-inference-server/server/issues/6320), which could be a great first issue. There's also [this protobuf issue](https://github.com/triton-inference-server/server/issues/6095). I also see some build issues under the "bug" label, and those could be good to work on, as you could iterate as you fix the build (one example is [this one](https://github.com/triton-inference-server/server/issues/5643) about making the werror flag configurable, though it could span multiple repos).
